pkg/cmd/generate/util.go

```diff
@@ -231,10 +232,15 @@ func sreLabels() map[string]string {
 	}
 }
 
-func sandboxSRENamespace(clusterType configuration.ClusterType) string {
-	sandboxSRENamespace := "sandbox-sre-host"
+func defaultSAsNamespace(kubeSawAdmins *assets.KubeSawAdmins, clusterType configuration.ClusterType) string {
 	if clusterType == configuration.Member {
-		sandboxSRENamespace = "sandbox-sre-member"
+		if kubeSawAdmins.DefaultServiceAccountsNamespace.Member != "" {
+			return kubeSawAdmins.DefaultServiceAccountsNamespace.Member
+		}
+		return "ksctl-member"
```

Comment:
Is there a reason why we'd want the default namespaces to be `ksctl-host` and `ksctl-member` instead of `kubesaw-host` and `kubesaw-member`?

Reply:
I chose `ksctl-` prefix as the content of the namespaces are is by `ksctl` only. `kubesaw-` prefix could confuse people as they could expect some operator or deployment running there.
We could name it: `kubesaw-admins-serviceaccounts-host`/`kubesaw-admins-serviceaccounts-member` to make it more explicit. Or short version: `kubesaw-admins-host`/`kubesaw-admins-member`

Reply:
If the namespace is only for admin SAs then `kubesaw-admins-host`/`kubesaw-admins-member` sounds good but if we'd add other non-admin SAs then maybe something like `kubesaw-serviceaccounts-host`/`kubesaw-serviceaccounts-member` fits. Or even `kubesaw-rbac-host`/`kubesaw-rbac-member`

Reply:
it's a default location for all ServiceAccounts defined in kubesaw-admins.yaml file. 
Let me rename it to `kubesaw-admins-host`/`kubesaw-admins-member`

Reply:
done 7f638ac

Reply:
@rajivnathan just wanted to check with you if the PR is OK now